tests/snuba/api/endpoints/test_organization_group_index.py

```diff
@@ -538,6 +539,77 @@ def test_assigned_to_pagination(self, patched_params_update):
 
         assert options.set("snuba.search.hits-sample-size", old_sample_size)
 
+    def test_seen_stats(self):
```

Comment:
I would add a test to make sure the skipped fields are handled as well.

Reply:
Will do